extensions/svgo/.prettierrc

```diff
@@ -1,4 +1,4 @@
 {
-  "printWidth": 120,
+  "printWidth": 80,
```

Comment:
Could we leave this to 120 as the rest of Raycast does

Reply:
I reverted the change